tests/client/connstring.rs

```diff
@@ -1,87 +1,81 @@
-use mongodb::connstring;
+use mongodb::connstring::{self, ConnectionString};
```

Comment:
Changes to this file are a mix of the moving of `connstring::parse` to `ConnectionString::parse` and from rustfmt.